docs/concept/TAP7.1-solution-design.md

```diff
@@ -358,6 +358,13 @@ Why this structure was proposed:
    architectural vision. It’s clear which domain each set of microservices belongs to (EDC for data exchange, identity
    for credential management, etc.). This clarity also helps new adopters navigate the ecosystem.
 
+| **Capability Bundle**     | **Default Components**                               | **Comments / Possible Replacements**                                             |
+|---------------------------|------------------------------------------------------|-----------------------------------------------------------------------------------|
+| **Identity Trust Bundle** | - ssi-dim-wallet-stub                                | Can be replaced by your own Identity Wallet.                                     |
```

Comment:
Please check the wording - not only with solution concept should be equal in the complete umbrelle HELM Chart repository.